### DISCUSSION TITLE
Torodd F. Ottestad - Zotero - kva, kvifor, korleis?

### DISCUSSION THREAD

**Discussion by giscus** [lektorodd/lektorodd.github.io#1] [General]
# Torodd F. Ottestad - Zotero - kva, kvifor, korleis?

https://lektorodd.github.io/posts/230127_zotero/

**Comment by lektorodd** [comment]
HVL har [ei god oversikt](https://www.hvl.no/bibliotek/skrive-og-referere/zotero-og-endnote/zotero/apa7th/) over ulike typar referansar i Zotero. Ta gjerne ein kikk der.

**Comment by PGelisa** [comment]
Jeg liker også koblingen med .ris og bibtex (eksempel ved å klikke på "Cite" på en side som denne https://www.sciencedirect.com/science/article/pii/S0029801823029700 eller denne https://www.idunn.no/doi/10.18261/naturen.147.4-5.2), da får man også all informasjon inn i Zotero.
